src/main/java/org/spongepowered/api/block/BlockSnapshotBuilder.java

```diff
@@ -63,7 +63,7 @@
     BlockSnapshotBuilder position(Vector3i position);
 
     /**
-     * Copies over block data from a {@link Location<World>}.
+     * Copies over block data from a {@link Location}<{@link World}>.
```

Comment:
fyi I did this with IDEA's Replace In Path — search: `\{@link (.*?)<(.*?)>\}`, replace: `{@link \$1}<{@link \$2}>`. apparently it uses `\$n` for backrefs.